winvnc/winvnc/DeskdupEngine.cpp

```diff
@@ -28,33 +53,20 @@ DeskDupEngine::DeskDupEngine()
 	HideCursorW8 = NULL;
 	osVer = osVersion();
 	if (osVer == OSWIN10) {
-#ifdef _X64
 		char szCurrentDir[MAX_PATH];

```

Comment:
As i told many times we use different names for the same dll when there is a risk that a x64 app can load a 32-bit dll.
We had to many bug requests in the past for this easy to avoid issue.